tox.ini

```diff
@@ -1,10 +1,11 @@
 [tox]
-envlist = {py27,py34,py35}-django{18,19}
+envlist = {py27,py34,py35}-django{18,19,110}
 
 [testenv]
 deps =
     pytest
     pytest-django
     django18: Django==1.8.10
     django19: Django==1.9.3
+    django110: Django==1.10.4
```

Comment:
I have a branch that takes care of most it if you want to add 1.11.1 and change the base to develop we can merge it.